.dev/docker/windows/x64-windows-rel.cmake

```diff
@@ -0,0 +1,4 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
```

Comment:
What's the purpose of these cmake files?

Reply:
To explicit set VCPKG to only build Release version of the libraries.

Reply:
Please add this as a comment on L12

Reply:
Please push this one alongside the --depth=1 changeset. Let the docker build.... (3 hours and going strong... Wish choco had vtk and goodies)

Reply:
Sure, I'm waiting to see if it completes before pushing again.